### ISSUE TITLE
Review guide

### ISSUE THREAD

**Issue by niravassar** [micronaut-guides/micronaut-cache-hazelcast#1]
Java version of the guide

**Comment by ilopmar** [comment]
I'm in the process of updating all Micronaut guides to 1.2.6 and also update the build files, build.gradle, latest dependencies,...

I have been reviewing the guide and modifying a few things to make it consistent with the changes I'm doing in the other guides. Please take a look at my commits.

I've noticed that the guide uses Micronaut 1.3.0.BUILD-SNAPSHOT and also the Hazelcast integration snapshot version. I think it would be better to wait until 1.3.0.M1 is out to publish the guide. Wdyt @sdelamo?

Things missing to publish the guide:
- Add Travis credentials. I've updated the scripts and enabled the build in Travis.
- Change the date of the publication (I've modified it to a future date so it's not published until we want).

**Comment by niravassar** [comment]
@ilopmar I looked at the commit. looks great. Let me know what else i can do to help.

**Comment by ilopmar** [comment]
@niravassar We need the input from @sdelamo because he is in charge of schedule and publish the guides. But for this one, as I mentioned I think we should wait until 1.3.0.M1 is released.

**Comment by alparslanavci** [comment]
@ilopmar do you have any updates on this guide?

**Comment by ilopmar** [comment]
No, I don't. @sdelamo needs to review it and schedule the publication. @niravassar can you please sync with Sergio so the guide gets published?
